sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1018,8 +1081,11 @@ def _df_getitem_unicode_idx_impl(self, idx):
 
         return _df_getitem_unicode_idx_impl
 
+    if isinstance(idx, types.Tuple):
```

Comment:
This should probably be a more strict check, e.g.
```suggestion

    if (isinstance(idx, types.Tuple)
        and all([isinstance(item, types.StringLiteral) for item in idx])):
```